VOKMockUrlProtocol.m

```diff
@@ -49,6 +49,12 @@ + (instancetype)containerWithResponse:(NSHTTPURLResponse *)response data:(NSData
 
 @implementation VOKMockUrlProtocol
 
+static NSBundle *testBundle = nil;
```

Comment:
how about setting `testBundle = [NSBundle bundleForClass:[self class]]` here instead of checking below when it's used?